ext/gd/tests/imageresolution_png.phpt

```diff
@@ -0,0 +1,42 @@
+--TEST--
+Set and get image resolution of PNG images
+--SKIPIF--
+<?php
+if (!extension_loaded('gd')) die('skip gd extension not available');
```

Comment:
The first test has a skip for supported image type, this one doesn't. Is there a reason for that ?